code/drasil-printers/lib/Language/Drasil/Markdown/Helpers.hs

```diff
@@ -0,0 +1,112 @@
+-- | Defines helper functions for creating Markdown files.
+module Language.Drasil.Markdown.Helpers where
+
+import Prelude hiding ((<>), lookup)
+import Text.PrettyPrint (Doc, text, empty, (<>), (<+>), ($$), hcat, vcat)
+import Data.List.Split (splitOn)
+import Data.Map (lookup)
+import Language.Drasil.Printing.Helpers (ast)
+import Language.Drasil.Printing.LayoutObj (RefMap)
+
+data Variation =  Id | Align
+
+-- | Curly braces.
+br :: Doc -> Doc
+br x = text "{" <> x <> text "}"
+
+-- | Square brackets
+sq :: Doc -> Doc
+sq t = text "[" <> t <> text "]" 
+
+-- | Parenthesis
+paren :: Doc -> Doc
+paren t = text "(" <> t <> text ")" 
+
+-- | Angled brackets
+ang :: Doc -> Doc
+ang t = text "<" <> t <> text ">"
+
+-- | Bold text
+bold :: Doc -> Doc
+bold t = ast <> ast <> t <> ast <> ast
+
+-- | Italicized text
+em :: Doc -> Doc
+em t = ast <> t <> ast
+
+li, ul :: Doc -> Doc
+-- | List tag wrapper
+li = wrap "li" []
+-- | Unordered list tag wrapper.
+ul = wrap "ul" []
+
+-- | helper for wrapping HTML lists
+wrap :: String -> [String] -> Doc -> Doc
+wrap a = wrapGen hcat Id a empty
+
+-- | Helper for setting up HTML tags
+wrapGen :: ([Doc] -> Doc) -> Variation -> String -> Doc -> [String] -> Doc -> Doc
```

Comment:
Can you please add some more documentation to this?

Reply:
Added in https://github.com/JacquesCarette/Drasil/pull/3823/commits/3db0792f58ac67a73664cbb1da3e768558f3f447. Additionally, I modified the type signature to remove an unnecessary parameter.